src/handle_request.js

```diff
@@ -137,4 +137,12 @@ function handleRequest(mockAdapter, resolve, reject, config) {
   }
 }
 
+function getEffectiveDelay(adapter, handler){
+  var delayPerRequest = 0;
+  if(handler.length === 8){
+    delayPerRequest = handler[7];
+  }
+  return adapter.delayResponse + delayPerRequest
```

Comment:
Instead of adding it to the global delay, I think it would be more intuitive if it overrode the global delay instead.

Reply:
That actually makes sense, I have adjusted.